src/rev_ai/models/asynchronous/caption_type.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+"""Enum for caption content types"""
+
+from enum import Enum
+
+
+class CaptionType(Enum):
+    SRT = "application/x-subrip"
+    VTT = "text/vtt"
+
+    @classmethod
+    def from_string(cls, status):
```

Comment:
is this an override of a default python method?

Reply:
No it is not. This was automatically added when I built actually. You can see the same in the job_status enum